roboteam_ai/src/control/positionControllers/PosController.cpp

```diff
@@ -63,7 +63,9 @@ double PosController::getAvoidBallDistance() const {
 void PosController::setAvoidBallDistance(double ballDistance) {
     customAvoidBallDistance = ballDistance;
 }
-
+void PosController::setListenToInterface(bool listenToInterface) {
```

Comment:
i do not like this. By conditionally listening to the interface, how can you know if editing a pid in the interface will actually work? this obscures PID tuning a lot and you will shoot yourself in the foot eventually with something like this.  (I recall we had this feature a few months ago and I explicitly removed it because of that)

Reply:
I have to agree with Lukas on this.

Reply:
I think, as you added the keeper pid in interface, this could be removed again?

Reply:
after discussion I understand it. I think a comment should be added here that we should be careful with this function, plus a test that this works properly